include/xsimd/math/xsimd_math_utils.hpp

```diff
@@ -0,0 +1,104 @@
+/***************************************************************************
+* Copyright (c) Johan Mabille, Sylvain Corlay, Wolf Vollprecht and         *
+* Martin Renou                                                             *
+* Copyright (c) QuantStack                                                 *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
```

Comment:
I think this file should live in the `types` subfolder instead of `math` to guarantee the non cyclic dependency math -> types

Reply:
This file is only included from the math folder, so I think it's the right place.